docs/quickstart.md

```diff
@@ -1,428 +1,590 @@
 # Redocly OpenAPI CLI quickstart guide
 
-## Before you start
+## Before you begin
 
-1. [Install](installation.md) the OpenAPI CLI tool.
+* Get yourself a [GitHub](https://github.com/) account.
+* Install [node.js](https://nodejs.org/en/) on your local computer.
 
-## Step 1 - Prepare your API definition file
+## Step 1 - Install OpenAPI CLI
 
-Once you installed the tool, let's dive into what you can do with it.
+[Follow these steps to install OpenAPI CLI](./docs/installation.md).
 
-For testing and demonstration purposes, we will use an example OpenAPI definition file (provided below). Save it to your computer as `petstore.yaml`.
+## Step 2 - Clone the openapi-starter project
 
-```yaml Example OpenAPI definition file
-openapi: "3.0.0"
+[openapi-starter](https://github.com/Redocly/openapi-starter) creates a local folder structure that you can use to manage your API definition either as a single file, or in multi-file format. It also places a basic API definition file in the `openapi` root folder for those of you who don't have a definition file but need one to start exploring OpenAPI CLI. You're welcome!
+
+If you have your own definition file, go right ahead and use it.
+
+Go [here](openapi-starter.md) to learn how to clone the `openapi-starter` project and learn more about how it works.
+
+## Step 3 - Prepare your API definition file
```

Comment:
suggestion: I think it would make more sense to continue using the example API definition from the openapi-starter repo here, instead of switching to a completely different, third-party API definition. I know you say users can choose whichever they want, but we've just spent an entire step explaining how to set up openapi-starter, only to seemingly abandon it in the next step.

Reply:
@skadinna - you would like me to abandon using openweathermap.yaml altogether and just use the example API definition throughout for all examples?

Reply:
I _think_ it would be easier for both the readers and the docs writers, and here's why:

- if we use it, we're telling the readers a cohesive story where all our stuff is more or less integrated. That way, they always know where to find the complete API definition - in the openapi-starter repo. Also, if they find something wrong in the API definition or want to propose some improvements to it, they could do it by opening an issue in the openapi-starter repo. No such option when we use someone else's API definition. If they ask us e.g. why some schema looks the way it looks in a particular example, we can't really answer that for someone else's API...
- continuing from the previous point, if we use an API definition that we fully control, docs writers always know who to ask for help, and they will also know when the API definition is updated. If we use someone else's API definition, the writers would either have to periodically check whether the original authors have updated something in the API definition, or just not care about the definition being up-to-date. 

Of course, this is just my opinion...I hope it makes at least some sense :sweat_smile: Might be good to wait for feedback from @adamaltman and @swapnilogale too, before you proceed with any changes.

Reply:
No, Ivana that makes _perfect_ sense. :) I only introduced the openweathermap.yaml as I know you guys were trying to move away from petstore! If Adam and Swapnil agree, then I will re-do the examples using openapi.yaml.

Reply:
I agree to use the openapi-starter repo definition. 👍

Reply:
Agree with what @skadinna said. It's easier to control expectations with our own API definition, rather than people grumble about something we have no control over. Also, it allows us to identify any shortcomings with our own API.